citf.go

```
@@ -19,24 +19,31 @@ type CITF struct {
 }
 
 // NewCITF returns CITF struct. One need this in order to use any functionality of this framework.
-func NewCITF(confFilePath string) CITF {
+func NewCITF(confFilePath string) (CITF, error) {
 	var environment environments.Environment
 	if err := config.LoadConf(confFilePath); err != nil {
 		// Log this here
-		fmt.Fprintf(GinkgoWriter, "error loading config file. Error: %+v", err)
+		// Here, we don't want to return fatal error since we want to continue
```

// Here, we don't want to return any error since we want to continue